lib/ui/widgets/layout_widgets.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+import '../../utils/locator.dart';
+import '../theme/theme.dart';
+
+class SectionBanner extends StatelessWidget {
+  final String title;
+  final Color bannerColor;
+  final Color titleColor;
+  final int fontSize;
+
+  const SectionBanner({
+    Key key,
+    @required this.title,
+    this.bannerColor,
+    this.titleColor,
+    this.fontSize
+  }) : super(key: key);
+  
+  @override
+  Widget build(BuildContext context) {
+
+    ConcordThemeData _appTheme = locator<ConcordThemeManager>().theme;
```

Comment:
Shouldn't this be `final`?
```suggestion
   final ConcordThemeData _appTheme = locator<ConcordThemeManager>().theme;
```